source/release_notes/4.4.9-10.rst

```diff
@@ -0,0 +1,28 @@
+.. _PSMDB-4.4.9-10:
+
+================================================================================
+*Percona Server for MongoDB* 4.4.9-10
+================================================================================
+
+:Date: October 7, 2021
+:Installation: `Installing Percona Server for MongoDB <https://www.percona.com/doc/percona-server-for-mongodb/4.4/install/index.html>`_
+
+Percona Server for MongoDB 4.4.9-10 is an enhanced, source available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB 4.4.9 Community Edition.
+It supports MongoDB 4.4.9 protocols and drivers.
+
+Release Highlights
+==================
+
+This release of MongoDB fixes the issue with the inconsistency created between the documents and the indexes. This could cause corrupted data during recovery from unexpected shutdown as the inconsistent checkpoint would be used for data recovery and prevent the node from starting due to the Duplicate Key error. This issue affects MongoDB 4.4.2 - 4.4.8.
```

Comment:
as long as we say that previous version are affected by the inconsistency bug, it would be great to mention that we recommend all users to upgrade to this version. And probably provide a link to the upgrade doc. What do you think?

Reply:
Agree, will add it

Reply:
We need to provide more information here regarding the seriousness and possible and even likely consequences of the data impacting bugs found in all versions of MongoDB 4.4.x except 4.4.1 and hopefully 4.4.9.

As such  here is a warning paragraph that details the bugs and their potential impacts as well warning about DuplicateKey errors and possible Data Corruption.

Here is paragraph (or more) with more information about those bugs as well as a recommended upgrade path.

Current 4.4.x Warnings and Bugs of Importance 
October 5, 2021


Beginning with MongoDB v4.4.2, several data impacting or corrupting bugs were introduced. Details are listed below. 
Please update to the latest available 4.4.x version as soon as possible.  The latest current version is 4.4.9. 

Percona Server for MongoDB 4.4.9-xxxx includes the upstream fixes these bugs.  

Additional Upgrade planning note - If you are running version 4.4.4 or below, please skip upgrading to version 4.4.8 as that has been found to trigger the DuplicateKey error most consistently. The 2 WT bugs are found in all versions of MongoDB 4.4.x except 4.4.1. Ideally, upgrade directly from version 4.4.1 to 4.4.9 to attempt to avoid those data impacting or corrupting bugs.


   
WT-7426 - After upgrade to v4.4.5, startups or restarts can trigger WiredTigers RTS bug which can corrupt page metadata causing documents on affected pages to become invisible to MongoDB. This can lead to temporary query incorrectness or more likely a fatal error and inability to restart . Affects only v4.4.5.

SERVER-58936 - Unique Index constraint violations possible - can cause duplicate data  - fixed in 4.4.8

WT-7995 - Checkpoint thread can read and persist inconsistent version of data to disk. Can cause Duplicate Key error on startup and prevent node from starting. Unclean shutdowns can cause data inconsistency within documents, deleted documents to still exist, incomplete query results due to lost or inaccurate index entries, and/or missing documents.  Affects versions 4.4.2 through 4.4.8 as well as 5.0.0 through 5.0.2. Upgrade to fixed version MongoDB 4.4.9 or 5.0.3 as soon as possible.

WT-7984 and associated Server Bug SERVER-60371

WT-7984 - Bug that could cause Checkpoint thread to omit a page of data. If server experiences an unclean shutdown an inconsistent checkpoint is used for recovery and causes data corruption. 
Fixed in v4.4.9. 
Requires validate command to be run and possible data remediation via complete initial sync.

SERVER-60371 - If previously upgraded to v4.4.8 then upgrade to 4.4.9, could still experience DuplicateKey error and Fatal assertion. Related to the 2 previous WT bugs. 
Requires validate command to be run and possible data remediation via complete initial sync.
 Currently under review.  https://jira.mongodb.org/browse/SERVER-60371